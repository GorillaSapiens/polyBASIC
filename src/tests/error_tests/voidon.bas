on void goto 10
10 print
