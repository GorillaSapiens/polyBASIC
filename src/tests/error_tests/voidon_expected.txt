ERROR: @1:4, UNHANDLED OPERATION ❮306❯
